book/src/specs/kimchi.md

```diff
@@ -1376,83 +1376,128 @@ You can find these operations under the [proof creation](#proof-creation) and [p
 A proof consists of the following data structures:
 
 ```rs
-#[derive(Clone)]
-pub struct LookupEvaluations<Field> {
+#[serde_as]
```

Comment:
Really great work!  This code looks so much simpler than the earlier versions where you split the enums and structs into chunked and non-chunked types.